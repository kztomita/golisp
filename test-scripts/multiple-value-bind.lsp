(print (multiple-value-bind (a b) (values 1 2)))
(print (multiple-value-bind (a b) (values 1 2) (list a b)))
(print (multiple-value-bind (a b) (values 1) (list a b)))
(print (multiple-value-bind (a b) (values 1 2 3) (list a b)))
(print (multiple-value-bind (a b) 1 (list a b)))

