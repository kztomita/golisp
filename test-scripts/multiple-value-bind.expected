
nil 
(1 2) 
(1 nil) 
(1 2) 
(1 nil) 
